src/components/static/ToggleSelection.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useState } from "react";
+import Arrows from "../../../public/faqArrows.svg";
+import Image from "next/image";
+
+const ToggleSelection = ({ title, children }) => {
+  const [state, setState] = useState(false);
+  return (
+    <div className="bg-bear-dark py-2">
+      <button
+        onClick={() => setState(!state)}
+        className="flex items-center justify-between w-full py-2 pr-8 text-white font-chonburi text-sm md:text-xl"
+      >
+        {title}
+        <Image
+          src={Arrows}
+          width={25}
+          className={`${state && "rotate-180"} duration-300`}
+        />
```

Comment:
Check out react-icons for the double arrows there and see if any of them look smoother.

https://react-icons.github.io/react-icons/search/#q=doublearrow

I think the current icon looks weird with the rotation since it doesn't seem rotating around its center of mass (a `-scale-y-100` looks smoother, change the rotation to flip if you keep the image). So just try out a few and pick whichever seem fine.

![bh-flip](https://github.com/bearhack-ucr/bearhack2024/assets/12877445/eadef525-7d33-47b1-805a-b8bc68b5f7bd)
![bh-rotate](https://github.com/bearhack-ucr/bearhack2024/assets/12877445/c711ff73-0c21-468f-bacc-763123f17356)

Reply:
Aw it doesn't seem the gifs are long enough for it to be animated, if you click on the images, they should open in a new tab and you can compare it without making the changes.